packages/theme/tools/fetch-figma-variables.ts

```diff
@@ -134,6 +134,17 @@ StyleDictionary.registerFilter({
   }
 });
 
+const mainIndex = (options?: ThemeOptions) => {
+  const postfix = options?.useFigmaStructure ? `Fs`: ``
+  return `
```

Comment:
Perhaps we can standardise this? Something like
```ts
type Organization = {id: string; name: string;}
const organizations: Organizations[] = [
   {id: 'atb', name: 'AtB'},
   {id: 'troms', name: 'Nfk'},
   {id: 'fram', name: 'FRAM'},
   {id: 'innlandet', name: 'Troms'},
   {id: 'nfk', name: 'Innlandet'}
]
```
and then `mainIndex` can return
```ts
organizations.map(organization => (
`export {default as ${organization.name}Themes${postfix}} from './${organization.id}-theme/theme';`
))
.join("\n")
```

Reply:
Done :)